airbyte-ci/connectors/pipelines/README.md

```diff
@@ -123,7 +123,7 @@ At this point you can run `airbyte-ci` commands.
 
 | Option                                  | Default value                   | Mapped environment variable   | Description                                                                                 |
 | --------------------------------------- | ------------------------------- | ----------------------------- | ------------------------------------------------------------------------------------------- |
-| `--no-tui`                              |                                 |                               | Disables the Dagger terminal UI.                                                            |
+| `--enable-dagger-run/--disable-dagger-run` |  `--enable-dagger-run``      |                               | Disables the Dagger terminal UI.                                                            |
```

Comment:
Nice! I was worried about merging airbyte CI and airbyte CI internal but this works